deploy/manifests/validation-webhook-configuration.yaml

```diff
@@ -1,13 +1,13 @@
 ---
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1beta1  # TODO: change to v1 when we no longer support k8s <1.16
```

Comment:
These are user facing manifests and scripts. Can we instead open a Github issue to track this and remove this TODO?
Having TODO in code is okay but having it here _could_ lead to confusion for end-users.
Not a blocker for merge but good to have.

Reply:
Done. #763